main.css

```diff
@@ -32,12 +32,29 @@ table tr:last-child p {
 }
 
 .summary {
-    padding-left: 2em;
     font-size: 80%;
     text-align: justify;
     text-justify: inter-word;
 }
 
+.summary, .slides {
+    padding-left: 2em;
+    vertical-align: middle;
+}
+
+.slides a:before {
+    display: inline-block;
+    width: 1.8em;
+    height: 2em;
```

Comment:
For my taste the pdf logo is a little too large.

Reply:
I pushed a new commit introducing a smaller logo. Is it better like this?

Reply:
LGTM. Thanks.